loader.go

```diff
@@ -59,104 +60,114 @@ func logger() *zap.Logger {
 
 // Start docker loader
 func (dockerLoader *DockerLoader) Start() error {
-	if !dockerLoader.initialized {
-		dockerLoader.initialized = true
-		log := logger()
-
-		dockerClients := []docker.Client{}
-		for i, dockerSocket := range dockerLoader.options.DockerSockets {
-			// cf https://github.com/docker/go-docker/blob/master/client.go
-			// setenv to use NewEnvClient
-			// or manually
+	if dockerLoader.initialized {
```

Comment:
Since I touched this function, this `if` statement was inverted to flatten the implementation.